bon-macros/src/builder/builder_gen/models.rs

```diff
@@ -85,6 +85,13 @@ pub(super) struct StartFn {
 
     /// Overrides the default generics
     pub(super) generics: Option<Generics>,
+
+    /// Preserve the original fn token span.
+    /// This determines the generated fn's span start
+    pub(super) orig_fn_token: syn::token::Fn,
+    /// Preserve the original brace token.
+    /// This determines the generated fn's span end
+    pub(super) orig_brace_tokens: syn::token::Brace,
```

Comment:
`orig` in the field name couples the `StartFn` model with the `fn`-based builder generation logic logically. Since this struct is reused between `struct`-based and `fn`-based builders lets keep it agnostic to the origin of the builder. I think it's enough to just strip the `orig_` prefix from field names here. Just `fn_token` and `brace_tokens` is fine as it severs as a "request" to use these tokens for the generated starting function.


```suggestion
    pub(super) fn_token: syn::token::Fn,
    /// Preserve the original brace token.
    /// This determines the generated fn's span end
    pub(super) brace_tokens: syn::token::Brace,
```

Reply:
I have removed the `orig_` prefix.

Reply:
Just realised I missed a few spots, will be fixing soon.

Reply:
Changed to an even simpler implementation, the code here is no longer needed